src/extensions/default/JavaScriptCodeHints/node/TernNodeDomain.js

```diff
@@ -40,10 +40,16 @@ var self = {
 
 var Tern = require("tern"),
     Infer = require("tern/lib/infer");
+    
+require("tern/plugin/requirejs");
+require("tern/plugin/doc_comment");
+require("tern/plugin/angular");
```

Comment:
Yes Pete. Without these, commonjs dependency analysis and js doc parsing for parameter type inference doesn't work.